app/helpers/dot_helper.rb

```diff
@@ -158,7 +158,10 @@ def agent_edge(agent, receiver)
       end
 
       block('digraph', 'Agent Event Flow') {
-        # statement 'graph', rankdir: 'LR'
+        layout ||= ENV['DIAGRAM_DEFAULT_LAYOUT'].presence
```

Comment:
If it isn't set, the graph statement is omitted and Graphviz (or Google Chart API) uses its default layout.  You may of course explicitly add ` || 'dot'`, though.